recipe/meta.yaml

```diff
@@ -67,72 +65,83 @@ source:
-    - readline          # [unix]
-    - sqlite            # [unix]
-    - tk                # [unix]
-    - xz                # [unix]
-    - zlib              # [unix]
+    # - nasm 2.11.06      # [win]
+    - bzip2             # [not win]
+    - sqlite
+    - xz                # [not win]
+    - zlib              # [not win]
+    - openssl
+    - readline          # [not win]
+    - tk                # [not win]
     - ncurses           # [linux]
-    - libffi            # [unix]
+    - libffi            # [not win]
   run:
-    - openssl           # [unix]
-    - readline          # [unix]
-    - sqlite            # [unix]
-    - tk                # [unix]
-    - xz                # [unix]
-    - zlib              # [unix]
+    - bzip2             # [not win]
+    - sqlite
+    - xz                # [not win]
+    - zlib              # [not win]
+    - openssl
+    - readline          # [not win]
+    - tk                # [not win]
     - ncurses           # [linux]
-    - libffi            # [unix]
-    #- vs2015_runtime    # [win]
+    - libffi            # [not win]
 
 test:
   commands:
     - python -V
-    - python3 -V            # [unix]
+    - python3 -V            # [not win]
     - 2to3 -h
     - pydoc -h
-    - python3-config --help  # [unix]
+    - python3-config --help  # [not win]
     - conda inspect linkages -p $PREFIX $PKG_NAME  # [not win]
     - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
     # Test for " -partition=none", can be removed after toolchain is dropped
-    - python -c "import sys; files = [f for f in sys.argv[2:] if ' -partition=none' in open(f, 'r').read()]; assert len(files) == 0, files" ${PREFIX}/lib/*/*.py  # [unix]
+    - python -c "import sys; files = [f for f in sys.argv[2:] if ' -partition=none' in open(f, 'r').read()]; assert len(files) == 0, files" ${PREFIX}/lib/*/*.py  # [not win]
+    # - python -c "import sysconfig; print(sysconfig.get_config_var('CC'))"
+    # -  _CONDA_PYTHON_SYSCONFIGDATA_NAME=_sysconfigdata_x86_64_conda_cos6_linux_gnu python -c "import sysconfig; print(sysconfig.get_config_var('CC'))"  # [linux64]
 
 about:
   home: http://www.python.org/
   license: PSF
   license_file: LICENSE
-  summary: 'General purpose programming language.'
+  summary: General purpose programming language
+  description: |
+    Python is a widely used high-level, general-purpose, interpreted, dynamic
+    programming language. Its design philosophy emphasizes code
+    readability, and its syntax allows programmers to express concepts in
+    fewer lines of code than would be possible in languages such as C++ or
+    Java. The language provides constructs intended to enable clear programs
+    on both a small and large scale.
+  doc_url: https://www.python.org/doc/versions/
+  doc_source_url: https://github.com/python/pythondotorg/blob/master/docs/source/index.rst
+  dev_url: https://docs.python.org/devguide/
 
 extra:
   recipe-maintainers:
     - jakirkham
     - jjhelmus
+    - mingwandroid
```

Comment:
Hey @isuruf looks like @mingwandroid is already here :smile:

Reply:
Yeah, but this is not merged yet. I added him on master.

Reply:
Ah OK. I was planning on merging this as soon as OS X passes.